src/agenda/style.ts

```diff
@@ -34,13 +34,13 @@ export default function styleConstructor(theme: Theme = {}) {
       alignItems: 'center',
       backgroundColor: appStyle.calendarBackground
     },
-    weekday: {
+    dayHeader: {
```

Comment:
Shouldn't we call it `daysHeader`? 
btw here and in other places.. 

```suggestion
    daysHeader: {
```

Reply:
It's a style that repeats on other components for the same part. I would like to keep it this way and not change on all files. The aim is to make it the default style and delete it eventually.